app/Helpers/Setting.php

```diff
@@ -26,13 +26,13 @@ public function get(string $key, $default = ''): string
         })->first();
 
         if ($setting) {
-            return $setting->value;
+            return $setting->value ?? $default;
```

Comment:
Sebaiknya bagian sini jangan return `$default` karena di bawah juga return `$default`
```suggestion
            return $setting->value;
```

Untuk mengijinkan return value null, kita atur return value methodnya saja boleh string atau null
```
public function get(string $key, $default = ''): string | null
```

Reply:
sudah disesuaikan